.github/workflows/ci-cd.yml

```
@@ -30,97 +30,54 @@ jobs:
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd="pg_isready -U local_user -d local_db"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=5
+    env:
+      DATABASE_URL: postgres://local_user:local_password@localhost:5432/local_db
+      PORT: 3001
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)